source/FreeRTOS_ND.c

```diff
@@ -1059,8 +1059,8 @@
                    {
                        size_t uxICMPSize;
                        BaseType_t xCompare;
-                       NetworkEndPoint_t * pxTargetedEndPoint = pxEndPoint;
-                       NetworkEndPoint_t * pxEndPointFound = FreeRTOS_InterfaceEndPointOnNetMask_IPv6( pxNetworkBuffer->pxInterface, &( pxICMPHeader_IPv6->xIPv6Address ), 2 );
```

Comment:
What MISRA violation lead to this name change?

Reply:
`misra_c_2012_rule_5_1`
Cant have two or more identifiers with more than 31 most significant chars same.